Atlas/frontend/src/components/CulturalHeritage/Add.js

```diff
@@ -46,13 +46,20 @@ const Page = ({ user, token, addCHInputs, addCHErrors, handleCHInputChange, addC
         <span className="atlas-button" onClick={ () => addCH(addCHInputs, token, imageUrl) }><PlusIcon /> Add Cultural Heritage</span>
       </FormGroup>
         <FormGroup>
+            {imageUrl.length!==0?
+                <img src={imageUrl[0].url} alt="Uploaded"></img>
+                    :
             <Dropzone
                 onDrop={handleDrop}
                 multiple
                 accept="image/*">
 
-                <p>Drop your files or click here to upload</p>
+                {imageUrl.length!==0 ?
+                    <p>Image added successfully</p>
+                        : <p>Drop your files or click here to upload</p> }
+
             </Dropzone>
+                }
         </FormGroup>
```

Comment:
It's *very* hard to read ternary operator usage like this. And also, please use spaces before and after binary operators
```javascript
{ imageUrl.length !== 0
  ? ( <img src={imageUrl[0].url} alt="Uploaded"></img> )
  : ( <Dropzone
        onDrop={handleDrop}
        multiple
        accept="image/*"
      >
        <p>Drop your files or click here to upload</p>
        { imageUrl.length !== 0
          ? ( <p>Image added successfully</p> )
          : ( <p>Drop your files or click here to upload</p> )
        }
      </Dropzone>
    )
}
```